specification/columns/tags.md

```diff
@@ -1,6 +1,6 @@
 # Tags
 
-The Tags column represents the set of tags assigned to [*tag sources*](#glossary:tag-source) that also account for potential provider-defined or user-defined tag evaluations.  Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges.
+The Tags column represents the set of tags assigned to [*tag sources*](#glossary:tag-source) that also account for potential provider-defined or user-defined tag evaluations. Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges. Tags may also be referred to by providers by other terms, such as labels or networks tags.
```

Comment:
```suggestion
The Tags column represents the set of tags assigned to [*tag sources*](#glossary:tag-source) that also account for potential provider-defined or user-defined tag evaluations. Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges. Tags may also be referred to by providers using other terms such as labels.
```

Network tags are networking configurations, not tags as we had discussed. If a provider decides to include them, there's no harm, but as a practitioner, I would not expect that networking or security configurations (e.g. security marks, which are in a similar category) to be included in my consumption data.
https://cloud.google.com/vpc/docs/add-remove-network-tags

There is definitely a possibility of other terms for tags/labels, but I have yet to find one other than "annotations" which has a broader definition and would include the network tags/security marks listed above. GCP does have system_labels, but I still think about those as labels with system specifying their origin (similar to project.labels specifying labels on the project).

Reply:
Thanks Karl - I read through the link you provided and I'm still a little confused by network tags. I can see that they are very explicitly related to networking and would understand why they would have less use in a billing report, but all of the language does seem to refer to them pretty explicitly as "tags".

Nonetheless, I think it's fair to just remove them from this list and let GCP decide whether they want to include them in their exports :)

Makes sense about other names - I'm good with your suggestion and committed it!